app/controllers/admin/health/scheduled_documents_controller.rb

```diff
@@ -0,0 +1,89 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Admin::Health
+  class ScheduledDocumentsController < HealthController
+    before_action :require_has_administrative_access_to_health!
+    before_action :require_can_manage_accountable_care_organizations!
+    before_action :set_scheduled_document, only: [:edit, :update, :destroy]
+
+    ALLOWED_DOCUMENT_CLASSES = [
+      'Health::ScheduledDocuments::EnrollmentDisenrollment',
+    ].freeze
+
+    def index
+      @scheduled_documents = scheduled_document_source.all
```

Comment:
Maybe an `active` scope on `Health::ScheduledDocuments::Base` so we can disable these over time but not lose the config.  Also, that would probably indicate an `inactive` tab eventually.